private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -83,12 +87,12 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.Type,
 		typeFlagName,
 		"",
-		`The full type name of the serialized message (like acme.weather.v1.Units).
+		`The full type name of the serialized message (like acme.weather.v1.Units) within the input.
 Alternatively, this can be a fully qualified path to the type without providing the source (like buf.build/acme/weather#acme.weather.v1.Units).`,
 	)
 	flagSet.StringVar(
 		&f.Input,
-		inputFlagName,
+		payloadFlagName,
 		"-",
 		fmt.Sprintf(
 			`The location to read the input message. Must be one of format %s.`,
```

Comment:
```suggestion
			`The location to read the payload. Must be one of format %s.`,
```
seems better to not call it input any more?

Reply:
yes